cmd/tstr/ctl_run_cmd.go

```diff
@@ -8,5 +8,7 @@ var ctlRunCmd = &cobra.Command{
 }
 
 func init() {
+	ctlTestCmd.PersistentFlags().StringVar(&ctlTestNamespace, "namespace", "", "The namespace to use.")
```

Comment:
Since the variable storing the namespace is now being used for non-test subcommand flags, we should probably rename `ctlTestNamespace` to just `ctlNamespace`. I'll address that in a follow up PR.